tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -1087,3 +1087,12 @@ test_i2c_tmp1075: tmp1075@98 {
 	consecutive-fault-measurements = <4>;
 	interrupt-mode;
 };
+
+test_i2c_ilps22qs: ilps22qs@99 {
+	compatible = "st,ilps22qs";
+	reg = <0x99>;
+	status = "okay";
+	odr = <LPS2xDF_DT_ODR_10HZ>;
+	lpf = <LPS2xDF_DT_LP_FILTER_ODR_4>;
+	avg = <LPS2xDF_DT_AVG_128_SAMPLES>;
```

Comment:
You could add a value for `fs`, as the sensor supports dual full-scale.

lps28dfw defines

`fs = <LPS28DFW_DT_FS_MODE_1_1260>; `

Reply:
Yes, I have already defined ILPS22QS_DT_FS_MODE_1_1260, so I can put it here.